README.md

```diff
@@ -31,152 +42,177 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
-  .transportationMode(TransportationMode()) // provide mode of transportation for better location enhancements
-  .delegate(self) // provide delegate to handle location updates locally if needed
-  .start()
+
+    .connection(ConnectionConfiguration(apiKey: ABLY_API_KEY, clientId: CLIENT_ID)) // provide Ably configuration with credentials
+
+    .log(LogConfiguration()) // provide logging configuration
+
+    .transportationMode(TransportationMode()) // provide mode of transportation for better location enhancements
+
+    .delegate(self) // provide delegate to handle location updates locally if needed
+
+    .start()
+    
 
 // Start tracking an asset
+
 publisher?.track(trackable: Trackable(id: trackingId)) // provide a tracking ID of the asset
+
 ```
 
-### Subscriber SDK 
+### Subscriber SDK
 
 The Asset Subscriber SDK can be used to receive location updates from a publisher in realtime. Here is an example of how Asset Subscribing SDK can be used:
 
 ```swift
+
 import AblyAssetTrackingSubscriber
 
 // Initialise a Subscriber
+
 subscriber = try? SubscriberFactory.subscribers() // get a Subscriber Builder
-  .connection(ConnectionConfiguration(apiKey: ABLY_API_KEY,
-                                      clientId: CLIENT_ID)) // provide Ably configuration with credentials
-  .trackingId(trackingId) // provide a Tracking ID for the asset to be tracked
-  .routingProfile(.cycling) // provide a routing profile for better location enhancements
-  .log(LogConfiguration()) // provide logging configuration
-  .delegate(self) // provide a delegate to handle received location updates
-  .start() // start listening to updates
+
+    .connection(ConnectionConfiguration(apiKey: ABLY_API_KEY, clientId: CLIENT_ID)) // provide Ably configuration with credentials
+
+    .trackingId(trackingId) // provide a Tracking ID for the asset to be tracked
+
+    .routingProfile(.cycling) // provide a routing profile for better location enhancements
+
+    .log(LogConfiguration()) // provide logging configuration
+
+    .delegate(self) // provide a delegate to handle received location updates
+
+    .start() // start listening to updates
+
 ```
 
 ### Authentication
 
-Both Subscriber and Publisher SDK support basic authentication (API key) and token authentication. Specify this by passing a `ConnectionConfiguration` to the Subscriber or Publisher builder: `SubscriberFactory.subscribers()
-  .connection(connectionConfiguration)`.
+Both Subscriber and Publisher SDK support basic authentication (API key) and token authentication. Specify this by passing a `ConnectionConfiguration` to the Subscriber or Publisher builder: `SubscriberFactory.subscribers().connection(connectionConfiguration)`.
+
+**To use basic authentication, set the following `ConnectionConfiguration` on the Subscriber or Publisher builder:**
 
-- To use basic authentication, set the following `ConnectionConfiguration` on the Subscriber or Publisher builder:
 ```swift
+
```

Comment:
Not sure why there's all this extra, spurious whitespace been added here either.

Reply:
fixed here 8d9502ef2d7a61158697bf9ea7e5602796959f65